app/Filament/Pages/AppSettingsPage.php

```diff
@@ -188,6 +219,55 @@ protected function getSearXngSettings(): Section
         );
     }
 
+    protected function testGotifyNotification(): void
+    {
+        $settings = $this->form->getState()['notification_services']['gotify'] ?? null;
+        
+        if (!$settings || empty($settings['url']) || empty($settings['token'])) {
+            Notification::make()
+                ->title('Error')
+                ->body('Please save your Gotify settings first')
+                ->danger()
+                ->send();
+            return;
+        }
+
+        try {
+            $url = rtrim($settings['url'], '/').'/message?token='.$settings['token'];
+            
+            $response = Http::post($url, [
+                'title' => 'Test Notification',
+                'message' => 'This is a test notification from PriceBuddy',
+                'priority' => 5,
+                'extras' => [
+                    'client::notification' => [
+                        'click' => url('/'),
+                    ],
+                ],
+            ]);
+            
+            $response->throw();
+            
+            Notification::make()
+                ->title('Success')
+                ->body('Test notification sent successfully')
+                ->success()
+                ->send();
+        } catch (\Illuminate\Http\Client\RequestException $e) {
```

Comment:
Generally prefer to have FQN simplified eg `use Illuminate\Http\Client\RequestException` then here just have `RequestException`